mne/source_estimate.py

```diff
@@ -1422,6 +1425,7 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
                                       clim=clim, cortex=cortex, size=size,
                                       background=background,
                                       foreground=foreground,
+                                      initial_time=initial_time,
```

Comment:
This was just added to `PySurfer`, right? So now people have to run `master` to do any brain plots? If so, that's not what we want -- you could do some introspection with `_get_args` and use `set_time_index` if the argument is not available in the constructor.